packages/stark-ui/src/modules/collapsible/components/_collapsible.component.scss

```diff
@@ -0,0 +1,56 @@
+/* ============================================================================== */
+/*  S t a r k   C o l l a p s i b l e                                             */
+/* ============================================================================== */
+/* stark-ui: packages/stark-ui/src/modules/collapsible/components/collapsible.component.scss */
+
+.stark-collapsible {
+  .mat-expansion-panel {
+	transition: box-shadow 150ms linear;
+	
+	&.mat-expanded {
+	  &.spin {
+		.mat-expansion-panel-header-title,
+		.mat-expansion-panel-header-description {
+		  .stark-collapsible-icon {
+			transform: rotate(180deg);
+		  }
+		}
+	  }
+	}
+	
+	.mat-expansion-panel-header {
+	  padding-right: 0;
+	}
+
+	.mat-expansion-panel-body {
+	  padding-top: 16px;
+	}
+
+	.mat-expansion-panel-header-description {
+	  justify-content: space-between;
+	  align-items: center;
+	  margin-right: 8px;
+	  .stark-collapsible-subtitle {
```

Comment:
In order to have consistency with fonts and material, we should use the predefined fonts, here it would be subheading-2, so this code will be:

```
font-size: mat-font-size($typography-config, subheading-2);
font-weight: mat-font-weight($typography-config, subheading-2);
line-height: mat-line-height($typography-config, subheading-2);
```